modules/migration-using-must-gather.adoc

```diff
@@ -39,7 +39,25 @@ endif::[]
 . Navigate to the directory where you want to store the `must-gather` data.
 . Run the `oc adm must-gather` command for one of the following data collection options:
 
-ifdef::troubleshooting-3-4,troubleshooting-mtc[]
+ifdef::troubleshooting-3-4[]
+* To collect data for the past hour:
++
+[source,terminal]
+----
+$ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v1.7
+----
++
+The data is saved as `must-gather/must-gather.tar.gz`. You can upload this file to a support case on the link:https://access.redhat.com/[Red Hat Customer Portal].
+* To collect data for the past 24 hours:
++
+[source,terminal]
+----
+$ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v1.7 -- /usr/bin/gather_metrics_dump
+----
++
+This operation can take a long time. The data is saved as `must-gather/metrics/prom_data.tar.gz`.
```

Comment:
I would remove the passive voice and make the second sentence more complete:

This command saves the data as the `must-gather/metrics/prom_data.tar.gz` file.

Reply:
If you do decide to take up my style suggestions, please also apply them to the currently-unchanged lines right after these ones, in order to ensure consistency between builds of this source.